sdk/eventhub/Azure.Messaging.EventHubs.Processor/api/Azure.Messaging.EventHubs.Processor.netstandard2.0.cs

```diff
@@ -1,30 +1,38 @@
 namespace Azure.Messaging.EventHubs
 {
-    public partial class EventProcessorClient
+    public partial class EventProcessorClient : Azure.Messaging.EventHubs.Primitives.EventProcessor<Azure.Messaging.EventHubs.Primitives.EventProcessorPartition>
     {
         protected EventProcessorClient() { }
         public EventProcessorClient(Azure.Storage.Blobs.BlobContainerClient checkpointStore, string consumerGroup, string connectionString) { }
         public EventProcessorClient(Azure.Storage.Blobs.BlobContainerClient checkpointStore, string consumerGroup, string connectionString, Azure.Messaging.EventHubs.EventProcessorClientOptions clientOptions) { }
         public EventProcessorClient(Azure.Storage.Blobs.BlobContainerClient checkpointStore, string consumerGroup, string connectionString, string eventHubName) { }
         public EventProcessorClient(Azure.Storage.Blobs.BlobContainerClient checkpointStore, string consumerGroup, string fullyQualifiedNamespace, string eventHubName, Azure.Core.TokenCredential credential, Azure.Messaging.EventHubs.EventProcessorClientOptions clientOptions = null) { }
         public EventProcessorClient(Azure.Storage.Blobs.BlobContainerClient checkpointStore, string consumerGroup, string connectionString, string eventHubName, Azure.Messaging.EventHubs.EventProcessorClientOptions clientOptions) { }
-        public string ConsumerGroup { get { throw null; } }
-        public string EventHubName { get { throw null; } }
-        public string FullyQualifiedNamespace { get { throw null; } }
-        public string Identifier { get { throw null; } }
-        public bool IsRunning { get { throw null; } protected set { } }
+        public new string ConsumerGroup { get { throw null; } }
+        public new string EventHubName { get { throw null; } }
+        public new string FullyQualifiedNamespace { get { throw null; } }
+        public new string Identifier { get { throw null; } }
+        public new bool IsRunning { get { throw null; } set { } }
```

Comment:
God... yes.   That's a definite mistake.  The property is intended to be `protected` with a settable override for testing and visibility shouldn't have been changed from the GA form.   I'll fix that in the next revision.

Side context:  Why a settable override rather than `virtual`?   We were concerned that the property is potentially used in situations where we didn't want to incur the performance hit for a virtual property lookup for a seldom used case.